site/specs/numbers.json

```diff
@@ -5523,92 +5523,89 @@
           },
           {
             "name": "tn",
-            "description": "<p><b class=\"required\">*</b> A Telephone Number (TN) that is referenced in the order</p>\n",
+            "description": "<b class=\"required\">*</b> A Telephone Number (TN) that is referenced in the order",
```

Comment:
@xxlaykxx it looks like a few of these fields are meant to be required by the description but are currently marked as `"required": false` in the spec. They should be marked as required and the * formatting may be removed at this point as the site will properly render this as a required field.

Reply:
fixed